earth_enterprise/src/fusion/autoingest/MiscConfigStorage.idl

```diff
@@ -40,6 +40,9 @@ class MiscConfigStorage {
   uint         PackGenAnalysisThreads = uint(1);
   GraphOpsType GraphOperations = MiscConfigStorage::NO_GRAPH_OPS;
 
+  // True to minify previous imagery and terrain levels, false to build from source imagery
```

Comment:
I should have realized this with my last suggestion, but at the end of this sentence it should probably refer to imagery and terrain.